### ISSUE TITLE
Feature request: allow styling an image with signature avatar and image

### ISSUE THREAD

**Issue by lirantal** [estruyf/screendown#4]
To provide an example, Ahmad Awais is a great content creator and he often shares tips or content through screenshots and you can see how he styles the footer with his avatar and his twitter handle:

![image](https://user-images.githubusercontent.com/316371/220628792-2ed2a2e0-0322-41d7-9ba7-99a07a8ac420.png)

It would be cool to be able to do the same and control the placement (and save as a preset so it doesn't require a setup every single time)

Source: https://twitter.com/MrAhmadAwais/status/1596149060070739971?s=20

**Comment by estruyf** [comment]
So a fig caption/watermark + image you can add on the image. Thanks @lirantal for the suggestion, will do some tests with it.

**Comment by lirantal** [comment]
Yep, exactly. I think this could really turn that into a killer feature because then folks can just use the extension from VS Code as they build, instead of copy/pasting code to things like Google Slides, ray.so, etc to apply the same graphics.

**Comment by estruyf** [comment]
![image](https://user-images.githubusercontent.com/2900833/220698464-fcec9c83-f7e9-4626-8327-3180d53d2f01.png)

**Comment by estruyf** [comment]
![image](https://user-images.githubusercontent.com/2900833/220708804-6d99b77e-3eb4-4d1d-bb4b-61f9f7a8429a.png)

Not complete yet, but getting there.

**Comment by estruyf** [comment]
![image](https://user-images.githubusercontent.com/2900833/220708930-4ed93454-f5fe-4697-9b23-477bd915c1a3.png)

**Comment by lirantal** [comment]
Omg this is so cool

**Comment by estruyf** [comment]
Think this will be the options:

![image](https://user-images.githubusercontent.com/2900833/220739091-02378b10-da3a-4f62-92bd-c61d3d624cc9.png)

**Comment by lirantal** [comment]
Is it possible to align the image a little more to the center of text (give it padding from the right side border of the image). We definitely want to make sure it looks visually aesthetic.

**Comment by lirantal** [comment]
Sharing some more social posts to give you an idea of how others are doing it:

![image](https://user-images.githubusercontent.com/316371/220742400-7fb14261-e0c1-44ec-80e0-922b40d6bd10.png)

![image](https://user-images.githubusercontent.com/316371/220742471-55639855-d583-453a-a1d1-e7b1328c4e45.png)

![image](https://user-images.githubusercontent.com/316371/220743227-677ca27e-089a-410d-ab9c-39f167dc1141.png)

![image](https://user-images.githubusercontent.com/316371/220743336-7d451b80-e312-4d23-a90e-23824abe10e7.png)

![image](https://user-images.githubusercontent.com/316371/220743395-bc530bb5-005b-4ea9-8d4b-376e907566f0.png)

**Comment by estruyf** [comment]
Ah, just published version `0.0.11` with the first implementation as seen above. For sure these things can be optimized.

Thanks for the examples.

**Comment by estruyf** [comment]
Better this way?

![image](https://user-images.githubusercontent.com/2900833/220752871-c6670440-a552-4727-80c9-6790b55df417.png)

![Screenshot 2023-02-22 at 21 36 17](https://user-images.githubusercontent.com/2900833/220752961-90a0dcec-4969-4922-b172-3fe9d07977c7.png)

**Comment by lirantal** [comment]
Yes, a whole different level 🔥

**Comment by estruyf** [comment]
@lirantal released version `0.0.12` let me know if you think anything needs to be improved.

**Comment by lirantal** [comment]
This is so frigging cool 😍

![image](https://user-images.githubusercontent.com/316371/220883255-6518bf02-53ac-4c9c-afce-aabb38f9c8e7.png)

**Comment by estruyf** [comment]
Looks amazing! Thanks @lirantal! 🙏

**Comment by estruyf** [comment]
This might make it even better

https://user-images.githubusercontent.com/2900833/220966048-bca3b319-5eaf-4172-b521-538596e6e614.mov

**Comment by lirantal** [comment]
Oooooh I haven't seen you can drag it like that! That's so cool

**Comment by estruyf** [comment]
Oh, I added it today, it was not available yesterday 😀
